ai_tor_tell_you_an_appointment/lib/sekai/GraphPage.dart

```diff
@@ -0,0 +1,261 @@
+                  height: 200, //200
+                  margin: const EdgeInsets.only(top: 15, left: 5, right: 5),
+                  child: Material(
+                    elevation: 4.0,
+                    borderRadius: BorderRadius.circular(24.0),
+                    shadowColor: Color(0x802196F3),
+                    child: Padding(
+                      padding: EdgeInsets.all(8.0),
+                      // child: SfCartesianChart()
+                      child: SfCartesianChart(
+                      //backgroundColor: StyleData.backgroundColor,
+                      //title: ChartTitle(text: thisChartType1.name), //graph name
+                      //legend: Legend(isVisible: true),                                        //icon บนกราฟ
+                      tooltipBehavior: TooltipBehavior(enable: true), //click to help user what it do
+                      series: <ChartSeries>[
+                        LineSeries<TimeData, DateTime>(
+                          name: 'Time',
+                          dataSource: _chartData,
+                          xValueMapper: (TimeData time, _) => time.dateTime,
+                          yValueMapper: (TimeData time, _) => time.time,
+                          dataLabelSettings:
+                              DataLabelSettings(isVisible: true), //แสดงเลขบนกราฟ
+                          enableTooltip: true,
+                          //color: StyleData.primaryColor
+                        )
+                      ],
+                      primaryXAxis: DateTimeAxis(), //แสดงชื่อของ bar นั้นๆ
+                      //primaryYAxis: NumericAxis(edgeLabelPlacement: EdgeLabelPlacement.shift,
+                      primaryYAxis: NumericAxis(
+                        minimum: 0, maximum: 1, interval: 0.5,
+                        edgeLabelPlacement: EdgeLabelPlacement.shift,
+                        numberFormat: NumberFormat.decimalPercentPattern(
+                            decimalDigits: 0), //ต้อง import init
+                        //title: AxisTitle(text: 'Success Rate'),
+                      ),
+                    ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+          bottomNavigationBar:
+              BottomNavigation(focused: BottomPages.statistics),
+          floatingActionButton: Builder(builder: (context) {
+            return FloatingActionButton(
+              child: const Icon(
+                Icons.view_day_outlined,
+              ),
+
+              onPressed: () =>
+                  Scaffold.of(context).openDrawer(), // <-- Opens drawer.
+            );
+          }),
+        );
+  }
+
+  List<TimeData> _getChartData(ChartType thisChartType1) {
+    List<TimeData> chartData = [];
+    DateTime now = DateTime.now();
+    if (thisChartType1 == ChartType.day) {
+        for (int i = 6; i >= 0; i--) {
+          DateTime date = now.subtract(Duration(days: i));
+          double value = values[i];
```

Comment:
Like we discussed earlier you should retrieve the data from another files. It may easier for somebody else to use the same data.